packages/vue/src/Charts/BarChart/OcBarChart.vue

```diff
@@ -19,7 +19,7 @@ const props = defineProps({
   yAxisFormatter: Function,
   xAxisFormatter: Function,
   tooltipFormatter: Function,
-  tooltipCurrency: String,
+  tooltipValueFormatter: Function,
```

Comment:
this need update in dashboard, if props already used

Reply:
I think we can create new prop without modify `tooltipCurrency`

Reply:
@kewcoder Yes, I know